lib/html/pipeline.rb

```diff
@@ -42,11 +42,15 @@ class Pipeline
     autoload :TableOfContentsFilter, 'html/pipeline/toc_filter'
     autoload :TextFilter,            'html/pipeline/text_filter'
 
-    # Our DOM implementation.
+    # Public: Our DOM implementation, a Nokogiri::HTML::DocumentFragment.
     DocumentFragment = Nokogiri::HTML::DocumentFragment
 
-    # Parse a String into a DocumentFragment object. When a DocumentFragment is
-    # provided, return it verbatim.
+    # Public: Parse a String into a DocumentFragment object. When a
+    #         DocumentFragment is provided, return it verbatim.
+    #
+    # document_or_html - The document or HTML to be parsed (default to '').
```

Comment:
Do we need to document the default that comes from code?

```ruby
document_or_html ||= ''
```

Or should we move the default to argument:

```ruby
def self.parse(document_or_html = '')
```